subprojects/core/src/main/java/org/gradle/api/internal/tasks/SnapshotTaskInputsBuildOperationResult.java

```diff
@@ -213,19 +213,7 @@ public void accept(CachingInputs inputs) {
     @Nullable
     @Override
     public Set<String> getInputPropertiesLoadedByUnknownClassLoader() {
-        return cachingState.getInputs()
-            .map(new java.util.function.Function<CachingInputs, Set<String>>() {
-                @Nullable
-                @Override
-                public Set<String> apply(CachingInputs inputs) {
-                    ImmutableSortedSet<String> invalidInputProperties = inputs.getNonCacheableInputProperties();
-                    if (invalidInputProperties.isEmpty()) {
-                        return null;
-                    }
-                    return invalidInputProperties;
-                }
-            })
-            .orElse(null);
+        return null;
```

Comment:
Is this always `null` now? If so, we don't need a method for this.

Reply:
This method is called by the scan plugin to capture the properties loaded by an unknown classloader. So we still need the method, though we should talk to the GE team after merging this what changes they want/need to do on their side to accommodate the new behavior.

Reply:
Ah, right. Got confused by this a bit, thinking it was one of the types we owned exclusively. Shall we deprecate the method?

Reply:
I deprecated the method.